src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,136 @@ Master Pre-Releases
+  `PR #8463 <https://github.com/pantsbuild/pants/pull/8463>`_
+
+* Fix tensorflow==1.14 globbing with platform-specific native_lib_names (#8378)
+  `PR #8378 <https://github.com/pantsbuild/pants/pull/8378>`_
+
+* Error out if the user explicitly tries to test a non-test tgt. (#8402)
+  `PR #8402 <https://github.com/pantsbuild/pants/pull/8402>`_
+
+* Revert "Format our python code with Black (#8391)" (#8409)
+  `PR #8391 <https://github.com/pantsbuild/pants/pull/8391>`_
+  `PR #8409 <https://github.com/pantsbuild/pants/pull/8409>`_
+
+New Features
+~~~~~~~~~~~~
+* Allow ignoring flake errors by name in addition to code. (#8444)
+  `PR #8444 <https://github.com/pantsbuild/pants/pull/8444>`_
+
+* Allow extra remote execution headers to be set (#8430)
+  `PR #8430 <https://github.com/pantsbuild/pants/pull/8430>`_
+
+* Start creating `binary` goal with support for python (#8399)
+  `PR #8399 <https://github.com/pantsbuild/pants/pull/8399>`_
+
+* Add support for running Pants with Python 3.8 (#8415)
+  `PR #8415 <https://github.com/pantsbuild/pants/pull/8415>`_
+
+Bugfixes
+~~~~~~~~
+* Send cumulative_timings & recorded_options in stats v2 too. (#8456)
+  `PR #8456 <https://github.com/pantsbuild/pants/pull/8456>`_
+
+* remove @_algebraic_data decorator to support @dataclass and mypy (#8437)
+  `PR #8437 <https://github.com/pantsbuild/pants/pull/8437>`_
+
+* Fix double check cache digest (#8416)
+  `PR #8416 <https://github.com/pantsbuild/pants/pull/8416>`_
+
+* Don't panic in create_scheduler (#8428)
+  `PR #8428 <https://github.com/pantsbuild/pants/pull/8428>`_
+
+* [junit-run] apply platform args to junit run java exec (#8421)
+  `PR #8421 <https://github.com/pantsbuild/pants/pull/8421>`_
+
+* Don't shade the nailgun package (#8418)
+  `PR #8418 <https://github.com/pantsbuild/pants/pull/8418>`_
+
+* Make subsystem dep explicit for RSC compile (#8406)
+  `PR #8406 <https://github.com/pantsbuild/pants/pull/8406>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Improve support for `Enum`s as option types (#8458)
+  `PR #8458 <https://github.com/pantsbuild/pants/pull/8458>`_
+  `Comment #8451 <https://github.com/pantsbuild/pants/pull/8451#discussion_r334075761>`
+
+* Port `test_pytest_run_integration.py` to V2 (#8466)
+  `PR #8466 <https://github.com/pantsbuild/pants/pull/8466>`_
+
+* Fix `PantsRunIntegrationTest.mock_buildroot()` to work with `--chroot` (#8465)
+  `PR #8465 <https://github.com/pantsbuild/pants/pull/8465>`_
+
+* Use f-strings in python tasks. (#8449)
+  `PR #8449 <https://github.com/pantsbuild/pants/pull/8449>`_
+
+* Run fuse tests on OSX again (#8453)
+  `PR #8453 <https://github.com/pantsbuild/pants/pull/8453>`_
+  `Issue #10017 <https://github.com/travis-ci/travis-ci/issues/10017>`_
+
+* Replace majority of remaining `datatype()`s with `@dataclass` (#8435)
+  `PR #8435 <https://github.com/pantsbuild/pants/pull/8435>`_
+  `PR #8423 <https://github.com/pantsbuild/pants/pull/8423>`_
```

Comment:
Don’t link to this

Reply:
Your commits tend to bring in lots of extraneous links like this...is that just due to your (fantastic) documentation habits?

Reply:
Hehe more me trying to break up enormous PRs into more palatable, bite-sized PRs. When I do this, I link them together so that people can figure out the connection.

Reply:
Got it. It confuses the hell out of the release script, but it isn't hard to fix so much as catch all the instances